src/types/transactions/encoded.ts

```diff
@@ -360,3 +340,33 @@ export interface EncodedLogicSig {
   sig?: Uint8Array;
   msig?: EncodedMultisig;
 }
+
+/**
+ * A structure for an encoded signed transaction object
+ */
+export interface EncodedSignedTransaction {
+  /**
+   * Transaction signature
+   */
+  sig: Buffer;
```

Comment:
```suggestion
  sig?: Buffer;
```
This field should be optional too. Only one of `sig`, `lsig`, or `msig` can be defined at once.

Reply:
got it.